testsuite/integration-arquillian/HOW-TO-RUN.md

```diff
@@ -717,6 +717,37 @@ The exact steps to configure Docker depend on the operating system.
 By default, the test will run against Undertow based embedded Keycloak Server, thus no distribution build is required beforehand.
 The exact command line arguments depend on the operating system.
 
+
+## Using DB Allocator Service
```

Comment:
This section belongs to https://github.com/keycloak/keycloak/blob/master/docs/tests-db.md

Reply:
Moved. Thanks!